scripts/installer.sh

```diff
@@ -41,11 +41,16 @@ main() {
 		#  - ID: the short name of the OS (e.g. "debian", "freebsd")
 		#  - VERSION_ID: the numeric release version for the OS, if any (e.g. "18.04")
 		#  - VERSION_CODENAME: the codename of the OS release, if any (e.g. "buster")
+		#  - UBUNTU_CODENAME: if it exists, as in linuxmint, use instead of VERSION_CODENAME
 		. /etc/os-release
 		case "$ID" in
 			ubuntu|pop|neon|zorin|elementary|linuxmint)
 				OS="ubuntu"
-				VERSION="$VERSION_CODENAME"
+				if [ "$UBUNTU_CODENAME" != "" ]; then
```

Comment:
You may need to use "${UBUNTU_CODENAME:-}" here, we set -u which will barf on undefined variables (i.e. on non-linux mint)

Reply:
Done. Tested on a non-linuxmint Ubuntu system as well.